src/shared/components/Button/Button.tsx

```diff
@@ -13,6 +13,7 @@ export type ButtonProps = {
   className?: string
   onClick?: (e: React.MouseEvent<HTMLButtonElement>) => void
   to?: To
+  type?: 'button' | 'submit'
```

Comment:
it would be nice to add `cursor: pointer` to all buttons with `type=submit`(because we will not use `onClick` in forms)

Reply:
Good point

Reply:
Done